solid-terms.ttl

```diff
@@ -215,9 +215,16 @@ solid:privateTypeIndex
     dc:issued "2018-01-24"^^xsd:date ;
     rdfs:comment "Points to an unlisted type index resource."@en ;
     rdfs:range solid:UnlistedDocument ;
-    rdfs:isDefinedBy <http://www.w3.org/ns/solid/terms#> ;
     rdfs:label "private type index"@en .
 
+solid:proxy
+    a rdf:Property ;
+    dc:issue "2024-10-10"^^xsd:date ;
+    rdfs:comment """A proxy endpoint for applications to use. The value of the property is intended to be used as the base URL for the request, e.g., the proxy as literal: `https://example.org/proxy?uri=` would be used to request `https://example.net/profile/card#me` as `https://example.org/proxy?uri=https://example.net/profile/card%23me`"""@en ;
```

Comment:
Alternative where the path is irrelevant, and what's most important is query parameter "proxy":

```suggestion
    rdfs:comment """A proxy endpoint for applications to use. The value of this property is intended to be used as the base URL for requests, following the URI Template: `{+path}{?proxy}`. For example, the proxy `https://example.org/foo?proxy=` can be used to fetch `https://example.net/profile/card` by making an HTTP request to `https://example.org/foo?proxy=https://example.net/profile/card`."""@en ;
```

I don't have a strong opinion on this besides it being dead simple for applications, essentially concatenating the resource they want to fetch to the proxy URL.

Reply:
It's not clear to me whether we should provide an URL-encoded uri (e.g. using `encodeURIComponent(originalURI))`, or not. This description seems to suggest non-encoded URIs, but I guess they may cause troubles.

Personally, I find the version before changes most clear; I guess that's a matter of personal preference.